datafusion/functions-aggregate/src/approx_percentile_cont_with_weight.rs

```diff
@@ -108,10 +156,8 @@ impl PartialEq<dyn Any> for ApproxPercentileContWithWeight {
         down_cast_any_ref(other)
             .downcast_ref::<Self>()
             .map(|x| {
-                self.approx_percentile_cont == x.approx_percentile_cont
-                    && self.column_expr.eq(&x.column_expr)
-                    && self.weight_expr.eq(&x.weight_expr)
-                    && self.percentile_expr.eq(&x.percentile_expr)
+                self.signature == x.signature
```

Comment:
I don't think we need this,  we should get `AggregateFunctionExpr`, and it has partialEq implemented already